web/modules/custom/eulogin/joinup_eulogin.permissions.yml

```diff
@@ -0,0 +1,4 @@
+unlimited access:
+  title: 'Unlimited access'
+  description: 'Allows unlimited access for user not linked to an EU Login account.'
+  restrict access: true
```

Comment:
This permission does not have a good name. It implies that the users with this permission are given unlimited access to ... something? But in fact the only thing happening is that their access to the EU account linking is being taken away from them. This is quite strange, that granting this permission is actually reducing the functionality for the user.

Wouldn't it be better to invert this, and call it something like `link account with eulogin`?